src/picknik_ur_studio_integration_testing/test/launch/test_ur5e_pinch_p_stop.test.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+# Copyright 2023 PickNik Inc.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+#    * Redistributions of source code must retain the above copyright
+#      notice, this list of conditions and the following disclaimer.
+#
+#    * Redistributions in binary form must reproduce the above copyright
+#      notice, this list of conditions and the following disclaimer in the
+#      documentation and/or other materials provided with the distribution.
+#
+#    * Neither the name of the PickNik Inc. nor the names of its
+#      contributors may be used to endorse or promote products derived from
+#      this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+
+
+import os
+import sys
+import unittest
+
+import launch_testing
+
+from picknik_ur_studio_integration_testing.generate_agent_plus_drivers_launch_description import (
+    generate_agent_plus_drivers_launch_description,
+)
+
+
+def generate_test_description():
+    return generate_agent_plus_drivers_launch_description(
+        gtest_name="test_ur5e_pinch_p_stop",
+        env_vars={
+            # Load site config for simulated UR-5e.
+            # Note: we want a configuration without the tool_changer_link for this one as it will throw off the collision check.
+            "STUDIO_CONFIG_PACKAGE": "picknik_001_ur5e_config",
```

Comment:
I think you missed changing this one too.

Reply:
No this one can't be changed it requires that config, sorry I commented and marked the conversation as resolved

Reply:
I don't see a comment? https://github.com/PickNikRobotics/moveit_studio_ws/pull/158#discussion_r1434059206

What's the reason for requiring this config? Because when we remove that from the core repo, this test won't be able to find the package.

But if there is something you plan to do in a follow-on PR to address this, that's OK and we can go ahead with this.

Reply:
Yes I also can't find the comment

```
            # Note: we want a configuration without the tool_changer_link for this one as it will throw off the collision check.
```

Reply:
ah I put it in the commit message https://github.com/PickNikRobotics/moveit_studio_ws/pull/158/commits/da3b1662e09a9917918d0dfb5ad5521882c582b5

Reply:
I see... hmm, well, we can deal with it in future.